internal/output/output.go

```diff
@@ -0,0 +1,61 @@
+package output
+
+import (
+	"strings"
+
+	"ldcli/internal/errors"
+)
+
+var ErrInvalidOutputKind = errors.NewError("output is invalid")
+
+// Outputter defines the different ways a command's response can be formatted based on
+// user input.
+type Outputter interface {
```

Comment:
We need an interface to define how to output JSON and and plain text versions of a response. This will be for a singular resource and when there are multiple resources.